solvers/Cov-tWDA.py

```diff
@@ -0,0 +1,61 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    from skorch.helper import to_numpy
+    from sklearn.pipeline import make_pipeline
+    from sklearn.pipeline import FunctionTransformer
+    from pyriemann.estimation import Covariances
+
+    from benchmark_utils.tWDA import tWDA
+
+
+class Solver(BaseSolver):

```

Comment:
Could you add a reference to your paper here in this style: 
https://github.com/benchopt/benchmark_lasso/blob/main/solvers/celer.py#L14

Reply:
@IA3005 could you add the reference so we can merge the PR?